drivers/hmis/app/graphql/types/hmis_schema/has_clients.rb

```diff
@@ -16,16 +16,16 @@ def clients_field(name = :clients, description = nil, type: [Types::HmisSchema::
           default_field_options = { type: type, null: false, description: description }
           field_options = default_field_options.merge(override_options)
           field(name, **field_options) do
-            argument :sort_field, Types::HmisSchema::ClientSortOption, required: false
+            argument :sort_order, Types::HmisSchema::ClientSortOption, required: false
             argument :sort_direction, Types::HmisSchema::SortDirection, required: false
             instance_eval(&block) if block_given?
           end
         end
       end
 
-      def resolve_clients(scope = object.clients, sort_field: :LastName, sort_direction: :asc, _user: current_user, no_sort: false)
+      def resolve_clients(scope = object.clients, sort_order: :LastName, sort_direction: :asc, _user: current_user, no_sort: false)
```

Comment:
I'd like to see `sort_order` encompass `sort_direction` as well.  So you'd pass in something like `p_t[:ProjectName].asc`

Reply:
then it could be super complex if need be. and below you'd just call `order(sort_order)`

Reply:
Ok, updating this so the client queries `organization(sortOrder: NAME_ASC)` or `organization(sortOrder: NAME_DESC)`  instead of having a separate sort direction param. As an explanation to @ZachCMP: based on what we've learned in the warehouse, having a separate direction doesn't always make sense. For example we are going to want to sort clients by "most recent enrollment" or "recently updated" etc where we don't need to support the reverse.

Reply:
@gigxz That makes good sense to me. As long as we keep the `_ASC`/`_DESC` convention we should be able to handle whatever UI treatment we need.